### PR TITLE
add sponsors list

### PR DESCRIPTION
**Pull request by PatrykLucka**
**- adds sponsors navigation item**
**- adds sponsor-list section with empty tiles linked to "Become a sponsor" section**
Please note that this section is copied from 2019 website, but it's places with different name of the section (sponsor-list) to differentiate it from the "Become a sponsor" section, which uses "sponsor-offer"

### CODE REVIEW

**Review by illia-stv — APPROVED** [review]

File: src/index.html [hunk 1]
```diff
@@ -361,29 +361,132 @@ <h3>Lunch & Coffee</h3>
...
-          <p class="sponsor-offer__text">
+          <p class="sponsor-list__text">
             Each year, our sponsors and partners help us keep tickets affordable, organize
             a&nbsp;great event and welcome you in their stands.
           </p>
 
-          <h3 class="sponsor-offer__subheading"><span>Platinum sponsor</span></h3>
-          <ul class="sponsor-offer__list">
-            <li class="sponsor-offer__list-item">
+          <h3 class="sponsor-list__subheading"><span>Platinum sponsor</span></h3>
```
Comment (naugtur) [inline src/index.html]: Update section titles to reflect offer PDF

File: src/index.html [hunk 2]
```diff
@@ -361,29 +361,132 @@ <h3>Lunch & Coffee</h3>
...
+            <li class="sponsor-list__list-item">
               <a href="https://tsh.io/" title="The Software House" target="_blank">
                 <img alt="The Software House" src="images/sponsors/platinum/tsh.svg" />
               </a>
             </li>
+            <li class="sponsor-list__list-item">
+              <a
+                href="#sponsor_offer"
+                class="sponsor-list__list-item--offer-link"
+                title="Become a sponsor"
```
Comment (naugtur) [inline src/index.html]: There can be only one main

File: src/index.html [hunk 3]
```diff
@@ -361,29 +361,132 @@ <h3>Lunch & Coffee</h3>
...
+              >
+                <span class="sponsor-list__small-offer-link">Become a sponsor</span>
+              </a>
+            </li>
+          </ul>
+
+          <h3 class="sponsor-list__subheading"><span>Partners</span></h3>
```
Comment (naugtur) [inline src/index.html:461]: Put siepomaga.pl and PCC (venue) in partners

File: src/index.html [hunk 4]
```diff
@@ -361,29 +361,132 @@ <h3>Lunch & Coffee</h3>
...
+            <li class="sponsor-list__list-item">
+              <a href="https://gruba.it/" title="Gruba IT" target="_blank">
+                <img src="images/partners/grubait.svg" atl="Gruba IT" />
+              </a>
+            </li>
+            <li class="sponsor-list__list-item">
+              <a
+                href="#sponsor_offer"
+                class="sponsor-list__list-item--offer-link"
+                title="Become a sponsor"
```
Comment (naugtur) [inline src/index.html]: not in partners sections

File: src/index.html [hunk 5]
```diff
@@ -136,7 +136,7 @@
                   >#sponsors</a
```
Comment (naugtur) [inline src/index.html:136]: More gap please  
Screenshot 2022-06-17 at 22-22-07 meet js Summit 2022

Comment (naugtur) [reply src/index.html:136]: While we're at it - unbreakable spaces please, so it's a single bar
Screenshot 2022-06-17 at 22-23-23 meet js Summit 2022

**Review by naugtur — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 2 above)
Comment (PatrykLucka) [reply src/index.html]: Is there already main sponsor? Should I remove this tile?

**Comment by naugtur** [comment]
Nie buduje mi się lokalnie. (yarn start)
Działało u Ciebie @PatrykLucka ?

**Comment by PatrykLucka** [comment]
Też przestało - poszedł fix @naugtur

**Pull request merged**